### PR TITLE
Support for the create-apps scope

### PR DESCRIPTION
**Pull request by pedro**

### CODE REVIEW

File: lib/identity/auth.rb [hunk 1]
```diff
@@ -129,6 +129,7 @@ class Auth < Sinatra::Base
           @scope = [
             "global",
             "identity",
+            "create-apps",
```
Comment (brandur) [inline lib/identity/auth.rb]: Mind just keeping this list alphabetized? Thanks!

File: views/clients/authorize.slim [hunk 2]
```diff
@@ -21,6 +21,8 @@
               li Read and write access to <strong>all</strong> of your account, apps and resources.
             - elsif scope == "identity"
               li Read-only access to your account information.
+            - elsif scope == "create-apps"
```
Comment (brandur) [inline views/clients/authorize.slim]: Same down here.

**Comment by brandur** [comment]
Left a few minor comments here, but +1 looks good. Thanks!

**Comment by pedro** [comment]
oops, fixed! thanks :}

**Comment by brandur** [comment]
Sweet! Pulling this in.

**Comment by brandur** [comment]
@pedro Should now be live.

**Comment by pedro** [comment]
Sweet! Testing the whole thing.
Thank you!

**Pull request merged**